app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsController.scala

```diff
@@ -92,16 +92,16 @@ class ReliefDetailsController @Inject() (
     page: (Form[A], Call) => P
   )(
     requiredPreviousAnswer: ReliefDetailsAnswers => Option[R],
-    redirectToIfNoRequiredPreviousAnswer: Call
+    redirectToIfNoRequiredPreviousAnswer: ReliefDetailsAnswers => Call
```

Comment:
you might find it easier if you change 
```scala
    requiredPreviousAnswer: ReliefDetailsAnswers => Option[R]
```
to 
```scala
    hasRequiredPreviousAnswer: ReliefDetailsAnswers => Boolean
```
The value of the previous answer is never used - we only care about whether or not the value is there or not

Reply:
Done